src/screens/MainApp/DiagnoseRequest/screens/Gallery/components/ListItem/styles.js

```diff
@@ -0,0 +1,40 @@
+import { StyleSheet } from 'react-native'
+import { scale } from 'react-native-size-matters/extend'
+
+export const SCREEN_WIDTH = scale(327)
+
+const styles = StyleSheet.create({
+  badge: {
+    position: 'absolute',
+    bottom: 0,
+    right: 0,
+    height: scale(24),
+    width: scale(28),
+    marginBottom: scale(1),
+    marginRight: scale(3),
+    alignContent: 'center',
+    backgroundColor: '#5ECE84',
+    borderTopLeftRadius: scale(4)
+  },
+  badgeText: {
+    fontWeight: 'bold',
+    textAlign: 'center'
```

Comment:
I'm concerned this `fontWeight: 'bold'` reverts the font to the default one (React Native problem regarding custom fonts), It may not be noticeable as the custom font is similar to the default one but it's definitely something to check for. 

This `bold` for `Red Hat Display` font only seems to appear in this case though. It is not reflected in the styleguide.

Feel free to either use the `Raleway bold` (`SubHeader`) or add a `Red Hat Display Bold`  with a new text component